app/views/categories/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<% if user_signed_in? %>
+  <%= content_tag :nav, class: "custom-nav" do %>
+    <%= content_tag :div, class: "custom-nav-content" do %>
+      <% if current_page?(categories_path) %>
+        <%= content_tag :button, id: "hamburger-btn", class: "custom-popup-toggle", type: "button" do %>
+          <%= content_tag :i, "", class: "fa fa-bars fa-3x hamburger" %>
+        <% end %>
+        <%= content_tag :h2, "CATEGORIES" %>
+        <%= content_tag :i, "", class: "fa fa-search fa-lg search" %>
+
+        <%= content_tag :div, id: "popup", class: "hidden custom-popup" do %>
+          <%= link_to "#", class: "custom-popup-close" do %>
+            <%= content_tag :i, "", class: "fa fa-times home-menu" %>
+          <% end %>
+          <ul>
+            <li><%= link_to "Home", root_path, class: "home-menu" %></li>
+            <li><%= button_to "Log out", destroy_user_session_path, method: :delete %></li>
+          </ul>
+        <% end %>
+      <% end %>
+    <% end %>
+    <%= content_tag :div, class: "custom-nav-bottom" do %>
+      <%= link_to "Most Recent", "#", class: "custom-nav-link" %>
+      <%= link_to "Most Ancient", "#", class: "custom-nav-link" %>
+    <% end %>
+  <% end %>
+<% end %>
+
+<section class="custom-section">
+  <ul class="custom-list">
+    <% @categories.each do |category| %>
+      <li class="custom-list-item">
+        <%= render partial: "categories/category", locals: { category: category } %>
+      </li>
+    <% end %>
+  </ul>
+  <%= link_to "New Category", new_category_path, class: "custom-button" %>
+</section>
```

Comment:
- [ ] [REQUIRED] Good job so far! I appraciate what you have done but your `button`("New Category", ...) and `navigation`("Most recent", ...), colors I mentioned below are not matching with [Behance Design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), please implement them 🙏🏼 

![image](https://github.com/HERMON-1995/PennyWise_Budget_app_capstone/assets/63915024/44379558-1dd8-44d3-9aac-914ef7dcb0fa)

![image](https://github.com/HERMON-1995/PennyWise_Budget_app_capstone/assets/63915024/38e0fce1-5dce-4c58-b370-5b039a2cf22d)

Reply:
Thank you I will make it the same as the design. 😄

Reply:
- [ ] [OPTIONAL] You have done a great job implementing the recommended changes by the previous reviewer, however, you need to make the UI more friendly.  Kindly take a look at each category of the provided Behance design and ensure to stick to the design. The white `div` for each category takes a width of 100%. It will be nice to implement these recommendations in order to have a better user interface.

<img width="390" alt="Screenshot 2023-07-06 at 23 12 04" src="https://github.com/HERMON-1995/PennyWise_Budget_app_capstone/assets/58692590/4286a279-b8d6-4376-bb2f-858b737d8b06">